### PR TITLE
fermikit: support for aarch64

### PR DESCRIPTION
**Pull request by t-nojiri**
In aarch64, we will modify it to use sse2neon.

I modified it with reference to the following PR.
https://github.com/spack/spack/pull/17473

### CODE REVIEW

File: var/spack/repos/builtin/packages/fermikit/package.py [hunk 1]
```diff
@@ -17,6 +17,12 @@ class Fermikit(MakefilePackage):
             submodules=True)
 
     depends_on('zlib')
+    depends_on('sse2neon', when='target=aarch64:')
+
+    patch('bwa-ksw_for_aarch64.patch', when='target=aarch64:')
+    patch('fermi2-ksw_for_aarch64.patch', when='target=aarch64:')
+    patch('seqtk-ksw_for_aarch.patch', when='target=aarch64:')
+    patch('trimadap-ksw_for_aarch.patch', when='target=aarch64:')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/fermikit/package.py]: Can we have a unique patch for the four file:
```python
patch('sse2neon.patch', when='target=aarch64:')
```
?

**Review by alalazo — CHANGES_REQUESTED** [review]
Overall LGTM. I would just require that we use a unique patch rather than one patch per file.

File: var/spack/repos/builtin/packages/fermikit/package.py (hunk 1 above)
Comment (t-nojiri) [reply var/spack/repos/builtin/packages/fermikit/package.py]: Merge into one patch file.

**Review by alalazo — APPROVED** [review]

**Comment by alalazo** [comment]
Thanks!

**Pull request merged**